### DISCUSSION TITLE
What does state_action do and do I need to configure it?

### DISCUSSION THREAD

**Discussion by alexsaas** [Koenkk/zigbee2mqtt#20866] [Devices]
I have a few lights and in the specific settings there is the point state_action.

1. Could someone please explain to me what this feature does?

2. The documentation says default is false. For me, however, a question mark is displayed between true and false. Should I still set false here manually?

I would appreciate if someone could help me. Thanks in advance.

**Comment by alexsaas** [comment]
Can nobody really help me?

**Comment by alexsaas** [comment]
@Koenkk could you please explain it to me, should i manualy set to false? The docs says this is the default. In my installation there is a question mark.

**Reply by Koenkk** [reply]
You  don't need to configure it

**Reply by alexsaas** [reply]
Okay, thanks for the feedback. Then I'll just leave it like that. The question mark unsettled me.
